flow_test/transform_tasks/http.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
Guessing also just copied in, will keep quite about comments

Reply:
@ciaranevans What is the issue here?